apps.awesim.org/apps/myjobs/templates/Basic_R_Parallel_Job_Owens/pbs_script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#PBS -l walltime=2:10:00
+#PBS -l nodes=2:ppn=28
+#PBS -j oe
+#PBS -N ondemand/sys/myjobs/basic_r_parallel
+
+#  A Basic R Parallel Job for the OSC Owens Cluster
+#  https://www.osc.edu/resources/available_software/software_list/r
+
+#
+# The following lines set up the R environment
+#
+ml openmpi/1.10
```

Comment:
Prefer using explicit `module load` to `ml` in case this is the first time that someone has seen `lmod` in action.